libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,43 +7,43 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix aria-label="Search books">
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
-  <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
+  <section class="book-grid">
+    <div class="book" data-testing="book-item" *ngFor="let book of (books$ | async)">
       <div class="book--title">
-        {{ b.title }}
+        {{ book.title }}
       </div>
       <div class="book--content">
         <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+          <img src="{{ book.coverUrl }}" alt="Cover photo of book from search list" />
```

Comment:
As in the above comment, cf. https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
Similarly here I made alt attribute empty